### PR TITLE
Feature/add test networks

### PR DESCRIPTION
**Pull request by cds-amal**
Allow truffle sample project to be deployed on rinkeby and ropsten testnets.

### CODE REVIEW

**Comment by cds-amal** [comment]
Deploying to rinkeby or ropsten depends on setting up the infura endpoints, as well as a mnemonic. It expects the same mnemonic for both testnets.

**Review by CruzMolina — APPROVED** [review]
✨

File: ui-tests/truffle/truffle-config.js [hunk 1]
```diff
@@ -7,6 +11,14 @@ module.exports = {
...
       network_id: '*'
-    }
+    },
+    ropsten: {
+      provider: () => new HDWalletProvider(mnemonic, ropsten_endpoint),
+      network_id: '3'
+    },
+    rinkeby: {
+      provider: () => new HDWalletProvider(mnemonic, rinkeby_endpoint),
+    network_id: '4'
```
Comment (eggplantzzz) [inline ui-tests/truffle/truffle-config.js]: indentation

File: ui-tests/truffle/truffle-config.js [hunk 2]
```diff
@@ -1,4 +1,8 @@
-// const path = require("path");
+const HDWalletProvider = require('@truffle/hdwallet-provider')
+
+const mnemonic = process.env['MNEMONIC']
+const rinkeby_endpoint = process.env['RINKEBY_ENDPOINT']
```
Comment (eggplantzzz) [inline ui-tests/truffle/truffle-config.js]: Not going to fight this but how do you feel about not using snake case for this and using camel case instead?

File: ui-tests/truffle/truffle-config.js (hunk 1 above)
Comment (cds-amal) [reply ui-tests/truffle/truffle-config.js]: *blush*

File: ui-tests/truffle/truffle-config.js (hunk 2 above)
Comment (cds-amal) [reply ui-tests/truffle/truffle-config.js]: a good argument

**Pull request merged**